.trunk/trunk.yaml

```diff
@@ -88,7 +88,7 @@ lint:
 actions:
   disabled:
     - trunk-announce
-    - trunk-check-pre-push
     - trunk-fmt-pre-commit
```

Comment:
fmt pre-commit could also be interesting no ?

Reply:
no, i find it really annoying in the dev flow , as you commit a lot more than you push, which interrupts the flow

Reply:
I see 
then a fmt pre push would be better

Reply:
no because for a fmt pre-push you need an automated commit system, which is annoying, and you can't always fix the check issues automatically - this just replicates what we had on the kakarot, which I think was good